src/TSMapEditor/Config/UI/Windows/ScriptsWindow.ini

```
@@ -25,6 +25,7 @@ $CC20=tbParameterValue:EditorNumberTextBox
 $CC21=btnEditorPresetValues:MenuButton
 $CC22=lblActionDescription:XNALabel
 $CC23=panelActionDescription:EditorPanel
+$CC24=btnEditorPresetValuesWindow:EditorButton
```

```suggestion
$CC21=btnEditorPresetValues:MenuButton
$CC22=btnEditorPresetValuesWindow:EditorButton
$CC23=lblActionDescription:XNALabel
$CC24=panelActionDescription:EditorPanel
```

Since this exists visually in the same space as `btnEditorPresetValues`, and the `[btnEditorPresetValuesWindow]` section has been placed under the `[btnEditorPresetValues]` section, it would be logical for this to also be defined right below `btnEditorPresetValues` in the list.